features/reservations/staff_comment_on_reservation.feature

```diff
@@ -9,18 +9,6 @@ Background: customer with existing reservation
 
   Given I am logged in as boxoffice manager
 
-Scenario: add comment to revenue reservation
```

Comment:
Why was this scenario deleted?

Reply:
I think we deleted this scenario to reflect the fact that order level comment no longer exists (if we understood this test correctly).

Reply:
and the new scenario we added checks item level comment.